src/CVSelect/ContentViewList.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  Button,
+  CardFooter,
+  TextInput,
+} from "@patternfly/react-core";
+import "@patternfly/react-core/dist/styles/base.css";
+
+import { AuthContext } from "../utils/AuthProvider";
+
+import "./ContentViews.css";
+
+export default function ContentViewList() {
+  const [contentViews, setContentViews] = React.useState([]);
+
+  const { basicAuth } = React.useContext(AuthContext);
+
+  React.useEffect(() => {
+    if (!basicAuth.basicAuth || !basicAuth.host) {
+      return;
+    }
+    fetch(`https://${basicAuth.host}/katello/api/content_views/`, {
+      method: "GET",
+      headers: {
+        Authorization: `Basic ${basicAuth.basicAuth}`,
+      },
+    })
+      .then((response) => response.json())
+      .then((arrayResponse) => {
+        setContentViews(arrayResponse.results);
+      });
+  }, [basicAuth]);
+
+  // eslint-disable-next-line no-unused-vars
```

Comment:
If you're not going to use the event, don't pass it through. You shouldn't need to disable eslint for this line.